trace/exporter_type.go

```diff
@@ -33,6 +33,16 @@ func (t ExporterType) MarshalJSON() ([]byte, error) {
 	return []byte(`"` + t.String() + `"`), nil
 }
 
+func (t *ExporterType) UnmarshalJSON(b []byte) error {
+	exporterTypeStr := strings.Trim(string(b), `"`)
```

Comment:
Should we be erroring if the quotes are formatted unexpectedly? For instance:
- `grpc`
- `"grpc`
- `""grpc`
- `""grpc"""`

would all be unmarshalled without issue, but non of them are formatted as expected.

Reply:
Done